types/raf/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// Type definitions for raf 3.4
+// Project: https://github.com/chrisdickinson/raf#readme
+// Definitions by: Ben Lorantfy <https://github.com/BenLorantfy>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare const raf: {
+  (callback: (timestamp: number) => void): number;
+  cancel: (handle: number) => void;
+  polyfill: (globalObject?: any) => void;
```

Comment:
Do you also need to add `raf/polyfill.d.ts` based on https://github.com/chrisdickinson/raf#rafpolyfillobject

Reply:
I'm not sure, it doesn't have any exports so I don't know if I need it? It would just be a blank file.